Dockerfile

```diff
@@ -28,7 +30,9 @@ RUN node_modules/bower/bin/bower install --allow-root
 
 RUN nosetests --all-modules --exe
 
-EXPOSE 5050
+RUN python3 run.py --config=instance_config.py --freeze --norun
```

Comment:
I think the config file specified should be `examples/iowa.py` so we don't have to have a new file.